content/en/blog/_posts/2018-05-29-announcing-kustomize.md

```diff
@@ -0,0 +1,245 @@
+source material makes it difficult to benefit from
+ongoing improvements to the source material. Taking
+this approach with many teams or environments, each
+with their own variants of a configuration, makes a
+simple upgrade intractable.
+
+Another approach to reuse is to express the source
+material as a parameterized template.  A tool processes
+the template—executing any embedded scripting and
+replacing parameters with desired values—to generate
+the configuration. Reuse comes from using different
+sets of values with the same template. The challenge
+here is that the templates and value files are not
+specifications of Kubernetes API resources. They are,
+necessarily, a new thing, a new language, that wraps
+the Kubernetes API. And yes, they can be powerful, but
+bring with them learning and tooling costs. Different
+teams want different changes—so almost every
+specification that you can include in a YAML file
+becomes a parameter that needs a value. As a result,
+the value sets get large, since all parameters (that
+don't have trusted defaults) must be specified for
+replacement. This defeats one of the goals of
+reuse—keeping the differences between the variants
+small in size and easy to understand in the absence of
+a full resource declaration.
+
+## A new option for configuration customization
+
+Compare that to **kustomize**, where the tool’s
+behavior is determined by declarative specifications
+expressed in a file called `kustomization.yaml`.
+
+The **kustomize** program reads the file and the
+Kubernetes API resource files it references, then emits
+complete resources to standard output. This text output
+can be further processed by other tools, or streamed
+directly to **kubectl** for application to a cluster.
+
+For example, if a file called `kustomization.yaml`
+containing
+
+> ```
+> commonLabels:
+>   app: hello
+> resources:
+> - deployment.yaml
+> - configMap.yaml
+> - service.yaml
+> ```
+
+is in the current working directory, along with
+the three resource files it mentions, then running
+
+```
+kustomize build
+```
+
+emits a YAML stream that includes the three given
+resources, and adds a common label `app: hello` to
+each resource.
+
+Similarly, you can use a *commonAnnotations* field to
+add an annotation to all resources, and a *namePrefix*
+field to add a common prefix to all resource
+names. This trivial yet common customization is just
+the beginning.
+
+A more common use case is that you’ll need multiple
+variants of a common set of resources, e.g., a
+*development*, *staging* and *production* variant.
+
+For this purpose, **kustomize** supports the idea of an
+*overlay* and a *base*. Both are represented by a
+kustomization file. The base declares things that the
+variants share in common (both resources and a common
+customization of those resources), and the overlays
+declare the differences.
+
+Here’s a file system layout to manage a *staging* and
+*production* variant of a given cluster app:
+
+> ```
```

Comment:
Remove >

Reply:
Are you saying don't indent the block?  Was doing that to avoid confusing these code blocks with commands that are meant to be copy pasted.

If one removes only the first and last indent, then all the greater than signs that remain are taken literally (they appear in the text).